specifications/api/swagger.yaml

```diff
@@ -270,15 +270,13 @@ x-walletPassphrase: &walletPassphrase
 
 x-walletPassphraseHash: &walletPassphraseHash
   description: |
-    A hash of master passphrase. The hash should be a 64-byte output of a Scrypt function with the following parameters:
+    A hash of master passphrase. The hash should be an output of a Scrypt function with the following parameters:
```

Comment:
We do know the size however right, it's actually a 100 bytes? So 200 hex-encoded. Isn't it.

Reply:
when looking at https://github.com/input-output-hk/cardano-wallet/pull/1476/files#diff-131f348ed80ae192131796acae9042efR247-R280
I see that the hex-encoded strings are the same in length, but the one added by you ("patate") is different. I used this to have it : 
```
let p = Passphrase @"raw" $ BA.convert $ T.encodeUtf8 PASSWD
hp <- encryptPasswordWithScrypt p
TR.trace ("hp:"<>show (hex hp))
```